compiler/lib/src/main/scala/codegen/CppWriter/EnumCppWriter.scala

```diff
@@ -355,12 +355,12 @@ case class EnumCppWriter(
       List.concat(
         List(
           linesClassMember(
-            lines("\n#if FW_SERIALIZABLE_TO_STRING"),
+            lines("\n#ifdef FW_SERIALIZABLE_TO_STRING"),
             CppDoc.Lines.Cpp
           )
         ),
         wrapClassMembersInIfDirective(
-          "\n#if FW_SERIALIZABLE_TO_STRING || FW_ENABLE_TEXT_LOGGING",
+          "\n#ifdef FW_SERIALIZABLE_TO_STRING || FW_ENABLE_TEXT_LOGGING",
```

Comment:
I think we should delete ` || FW_ENABLE_TEXT_LOGGING` here. If `FW_SERIALIZABLE_TO_STRING` is off and `FW_ENABLE_TEXT_LOGGING` is on, then the build is broken. I confirmed this with a quick test.

Reply:
good catch, I just removed it

Reply:
Thanks! On further consideration, I reverted this change. I think the intent of the or condition is to enable display of enums in text logging even when serializable-to-string is turned off.

Reply:
However, now I see this in the F Prime config code:

https://github.com/nasa/fprime/blob/af76f089e5aaee774d129e9005e19aaeab36e2f0/Fw/Cfg/ConfigCheck.cpp#L39-L42

This suggests that `FW_ENABLE_TEXT_LOGGING != 0` implies `FW_SERIALIZABLE_TO_STRING != 0`. So maybe we should revert the reversion.